hadoop-ozone/ozone-manager/pom.xml

```diff
@@ -69,6 +69,12 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <artifactId>ozone-interface-storage</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.ozone</groupId>
+      <artifactId>ozone-s3-secret-store</artifactId>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
> Thanks for review. I have fixed your comment. Could you please take a look again?

@Pochatkin Thanks for updating the patch.  It is much improved, resolving several previous comments.

However, `ozone-manager` still depends on Vault, only transitively.
```
[INFO] org.apache.ozone:ozone-manager:jar:1.4.0-SNAPSHOT
...
[INFO] +- org.apache.ozone:ozone-s3-secret-store:jar:1.4.0-SNAPSHOT:compile
[INFO] |  \- com.bettercloud:vault-java-driver:jar:5.1.0:compile
```

We need an interface.  Originally I suggested this to be a separate module, but can also be in `ozone-manager` for simplicity.  Vault-based implementation should depend on the module that contains the interface (separate module or `ozone-manager` depending on your choice for the above), not the other way around.

Reply:
We already have some configuration properties where the implementation is pluggable and is instantiated via reflection, see callers of `ConfigurationSource#getClass`.

Reply:
Thanks @Pochatkin for updating the patch.  Can you please fix checkstyle/findbugs/rat failures?

https://github.com/Pochatkin/ozone/actions/runs/4499127644

Also, I think the new module should be added as a dependency to `hadoop-ozone/dist`, so that it ends up in the release tarballs.  (With that I think the failed _dependency_ check will be passing again.)